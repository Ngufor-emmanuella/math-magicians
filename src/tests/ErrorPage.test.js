import React from 'react';
import renderer from 'react-test-renderer';
import NotFoundPage from '../components/ErrorPage';

test('should match the snapshot', () => {
  const makeNew = renderer.create(<NotFoundPage />);
  const tree = makeNew.toJSON();
  expect(tree).toMatchSnapshot();
});
