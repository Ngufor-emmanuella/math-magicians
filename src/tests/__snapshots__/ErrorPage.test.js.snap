// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<div
  className="error"
>
  <h2>
    404 Error....
  </h2>
  <h3>
    Page Not Found !!
  </h3>
</div>
`;
